raiden/transfer/state.py

```diff
@@ -325,7 +326,8 @@ def from_dict(cls, data: typing.Dict[str, typing.Any]) -> 'ChainState':
 
         restored = cls(
             pseudo_random_generator=pseudo_random_generator,
-            block_number=int(data['block_number']),
+            block_number=typing.BlockNumber(
```

Comment:
Here and in many other places in this file -- closing parentheses should go in their own line

Reply:
Done.